packages/maven.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Install a custom Maven version - https://maven.apache.org
+#
+# To run this script on Codeship, add the following
+# command to your project's setup commands:
+# \curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/maven.sh | bash -s
+#
+# Add the following environment variable to your project configuration
+# (otherwise the default below will be used).
+# * MAVEN_VERSION
```

Comment:
I would add a warning to only use 3.x since Apache doesn't offer versions 2 for download any longer, http://apache.mesi.com.ar/maven/maven-2/